apps/good_practice/types.py

```diff
@@ -156,6 +171,7 @@ class GoodPracticeFilterChoiceType:
     tags: Optional[List[TagType]]
     start_year: int
     end_year: int
+    success_factor: Optional[List[SuccessFactorType]]
```

Comment:
Let's return with empty array if empty
```suggestion
    success_factor: List[SuccessFactorType]
```